pkg/features/kube_features.go

```diff
@@ -850,6 +845,5 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	// features that enable backwards compatibility but are scheduled to be removed
 	// ...
-	HPAScaleToZero:         {Default: false, PreRelease: featuregate.Alpha},
-	LegacyNodeRoleBehavior: {Default: true, PreRelease: featuregate.Beta},
+	HPAScaleToZero: {Default: false, PreRelease: featuregate.Alpha},
```

Comment:
Wait, this feature is not related to the PR. Why is this being moved back to Alpha?